user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+exported to a .CSV file, and define other settings. For some of them, it is also possible to add new fields, as 
+described in the :ref:`Entity Fields guide <user-guide-field-management>`. 
+
+These actions are only available if the *"Manage Configurable Entities"* capability is set to *"System"*. 
+
+They are usually performed by the system administrators.
+
+
+.. _admin-capabilities-org-calendar-events: 
+
+Manage Organization Calendar Events
+-----------------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in organization-wide calendars, 
+which are described in more detail in the :ref:`corresponding section <user-guide-calendars-system>` of 
+the *Calendars Overview* guide.
+
+Organization calendar events are usually managed by organization-level managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add 
+     them to their own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage System Calendar Events
+-----------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in system-wide calendars, which 
+are described in more detail in the <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+System calendar events are usually managed by the company managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add them to their 
+     own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendars: 
+
+Manage System Calendars
+-----------------------
+
+
+If this capability has been set to *"System"*, the users can 
+:ref:`create <user-guide-calendars-system>` and :ref:`manage <user-guide-calendars-manage>` system-wide calendars.
+
+System-wide calendars are usually created and managed by system administrators and top managers.
+
+
+.. _admin-capabilities-passwords:
+ 
+Manage Users' Passwords
+-----------------------
+
+If the capability is set to *"System"*, the user can change passwords of other users. Usually, this is only done
+by system administrators when :ref:`creating or editing a user record <user-management-users>`. 
+
+.. hint::
+
+    This capability does not influence a user's ability to :ref:`edit their own 
+    password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge Entities
+--------------
+
+If the capability is set to *"System"*, the user can :ref:`merge <user-guide-ui-components-grids-delete-merge>` 
+several records of the same entity. By default the only entity, records of which can be merged is 
+:ref:accounts <user-guide-accounts>, but if other mergebale entities are added the setting will be applied. 
```

Comment:
Comment: The last sentence is a little confusing and should be rewritten. For instance, is "By default the only entity," supposed to be set off with that comma, or is the comma misplaced? (I.e., should it be "entity records"?) Also, it's a bit unclear what the sentence is saying. Is it that the only record that can be merged are accounts records? And what setting will be applied if other entities are added?